src/qt/walletcontroller.cpp

```diff
@@ -340,7 +351,11 @@ LoadWalletsActivity::LoadWalletsActivity(WalletController* wallet_controller, QW
 
 void LoadWalletsActivity::load()
 {
-    showProgressDialog(tr("Loading wallets…"));
+    showProgressDialog(
+        //: Title of progress window when which is displayed when wallets are being loaded.
```

Comment:
Just to be sure, is the first "when" correct?

```suggestion
        //: Title of progress window which is displayed when wallets are being loaded.
```
?

Reply:
Oops! It shouldn't have been there. Let me quickly correct it!

Reply:
Done Corrected! f86fe19